features/support/env.rb

```diff
@@ -10,4 +10,9 @@ def self.deed_api
     (ENV['DEED_API_URL'] ||
         'http://0.0.0.0:9020')
   end
+
+  def self.borrower_frontend
+    (ENV['BORROWER_FRONTEND_URL'] ||
+        'http://10.10.10.10:9030/searchdeed?')
```

Comment:
Specifying a full url like this as an environment variable limits it's reuse. Better practice would be to just specify the borrower frontend as the base url http://10.10.10.10:9030. Then when you want to navigate to the search deed page use visit(Env.borrower_frontend + '/searchdeed?'. This way if you wish to navigate to another page on the frontend you can use the env variable again as Env.borrower_frontend + <page_name> if that makes sense.